src/esp/gfx/RenderCamera.cpp

```diff
@@ -207,10 +207,12 @@ esp::geo::Ray RenderCamera::unproject(const Mn::Vector2i& viewportPosition,
       2 * Magnum::Vector2{viewPos} / Magnum::Vector2{viewport()} -
           Magnum::Vector2{1.0f},
       1.0};
+  const auto projMat = projectionMatrix();
 
   // compute the far plane distance
-  auto farDistance =
-      projectionMatrix()[3][2] / (1.0f + projectionMatrix()[2][2]);
+  // If projMat[3][3] == 0 then perpsective, otherwise ortho
+  auto farDistance = (projMat[3][3] == 0 ? projMat.perspectiveProjectionFar()
```

Comment:
(Annoying mode enabled) 

```suggestion
  const float farDistance = (projMat[3][3] == 0 ? projMat.perspectiveProjectionFar()
```